gooddata-sdk/gooddata_sdk/catalog/workspace/entity_model/filter_view.py

```diff
@@ -0,0 +1,147 @@
+            return self.relationships.user["data"].id
+        return None
+
+    @property
+    def analytical_dashboard_id(self) -> str | None:
+        if self.relationships and self.relationships.analytical_dashboard:
+            return self.relationships.analytical_dashboard["data"].id
+        return None
+
+    def assign_user(self, user_id: str) -> None:
+        if self.relationships is None:
+            self.relationships = CatalogFilterViewRelationships.create_user_analytical_dashboard_relationship(
+                user_id=user_id
+            )
+        else:
+            self.relationships.user = _data_entity(CatalogEntityIdentifier(id=user_id))
+
+    def assign_analytical_dashboard(self, analytical_dashboard_id: str) -> None:
+        if self.relationships is None:
+            self.relationships = CatalogFilterViewRelationships.create_user_analytical_dashboard_relationship(
+                analytical_dashboard_id=analytical_dashboard_id
+            )
+        else:
+            self.relationships.analytical_dashboard = _data_entity(CatalogEntityIdentifier(id=analytical_dashboard_id))
+
+    def clean_assignments(self) -> None:
+        if self.relationships is not None:
+            self.relationships.user = None
+            self.relationships.analytical_dashboard = None
+
+
+@attr.s(auto_attribs=True, kw_only=True)
+class CatalogFilterViewAttributes(Base):
+    content: dict[str, Any]
+    title: str
+    are_relations_valid: Optional[bool] = None
+    description: Optional[str] = None
+    is_default: Optional[bool] = None
+    tags: Optional[list[str]] = None
+
+    @staticmethod
+    def client_class() -> type[JsonApiFilterViewInAttributes]:
+        return JsonApiFilterViewInAttributes
+
+
+@attr.s(auto_attribs=True, kw_only=True)
+class CatalogFilterViewRelationships(Base):
+    user: Optional[dict[str, CatalogEntityIdentifier]] = None
+    analytical_dashboard: Optional[dict[str, CatalogEntityIdentifier]] = None
+
+    @staticmethod
+    def client_class() -> type[JsonApiFilterViewInRelationships]:
+        return JsonApiFilterViewInRelationships
+
+    @classmethod
+    def create_user_analytical_dashboard_relationship(
+        cls, user_id: Optional[str] = None, analytical_dashboard_id: Optional[str] = None
+    ) -> CatalogFilterViewRelationships | None:
+        if user_id is None and analytical_dashboard_id is None:
+            return None
+        assignee_user = _data_entity(CatalogEntityIdentifier(id=user_id, type="user")) if user_id else None
+        assignee_analytical_dashboard = (
+            _data_entity(CatalogEntityIdentifier(id=analytical_dashboard_id, type="analyticalDashboard"))
+            if analytical_dashboard_id
+            else None
+        )
+        return cls(user=assignee_user, analytical_dashboard=assignee_analytical_dashboard)
+
+
+@attr.s(auto_attribs=True, kw_only=True)
+class CatalogEntityIdentifier(Base):
+    id: str
+    type: Optional[str] = None
```

Comment:
I am not sure if CatalogEntityIdentifier is a good idea. There are identifiers in identfier.py Could we reuse them?